samples/apps/logging/logging.cpp

```diff
@@ -1198,11 +1198,16 @@ namespace loggingapp
 
 namespace ccfapp
 {
-  // SNIPPET_START: rpc_handler
+  // SNIPPET_START: app_interface
   std::shared_ptr<ccf::RpcFrontend> get_rpc_handler(
     ccf::NetworkTables& nwt, ccfapp::AbstractNodeContext& context)
   {
     return make_shared<loggingapp::Logger>(nwt, context);
   }
-  // SNIPPET_END: rpc_handler
+
+  std::vector<ccf::js::FFIPlugin> get_js_plugins()
+  {
+    return {};
+  }
```

Comment:
I'd really like to avoid requiring this extra bit of boilerplate in every app, if possible. I think we can do that with a weak symbol?

Reply:
I was just thinking the same, after fixing more apps...

Reply:
Done.